Source/JavaScriptCore/wasm/WasmSectionParser.cpp

```diff
@@ -790,9 +790,10 @@ auto SectionParser::parseFunctionType(uint32_t position, RefPtr<TypeDefinition>&
 
     uint32_t returnCount;
     WASM_PARSER_FAIL_IF(!parseVarUInt32(returnCount), "can't get ", position, "th Type's return count");
+    WASM_PARSER_FAIL_IF(returnCount > maxFunctionReturns, position, "th return count is too big ", returnCount, " maximum ", maxFunctionReturns);
```

Comment:
"th" always bothered me can we change these to`"return count of Type at index ", position, " is too big ", ...` and `"can't get Type's return count at index", position`

Reply:
Sounds good, changed.

Reply:
For now, in this change, I'll change arguments and return types, since there are too many of them.